tests/integ/test_data_source_api.py

```diff
@@ -348,3 +358,41 @@ def assert_datasource_statement_params_run_query(*args, **kwargs):
             single=True,
         )
         assert called == 2
+
+
+@pytest.mark.parametrize(
+    "custom_schema",
+    [
+        "id INTEGER, int_col INTEGER, real_col FLOAT, text_col STRING, blob_col BINARY, null_col STRING",
```

Comment:
I think we also need test of custom_schema that include all types

Reply:
updated